metadata-ingestion/src/datahub/cli/migration_utils.py

```diff
@@ -168,5 +273,20 @@ def get_incoming_relationships_dataset(urn: str) -> Iterable[Dict]:
             "Produces",
             "ForeignKeyToDataset",
             "DerivedFrom",
+            "IsPartOf",
+        ],
+    )
+
+
+def get_outgoing_relationships_dataset(urn: str) -> Iterable[Dict]:
```

Comment:
suffixing in _dataset is confusing, given that "dataset" also means something special to datahub :) Consider renaming this to something that won't easily be confused